src/ProblemReductions.jl

```diff
@@ -29,7 +29,9 @@ export QUBO
 # rules
 export target_problem, AbstractProblem, AbstractReductionResult, reduceto, extract_solution, reduction_complexity
 export LogicGadget, truth_table
+
 export ReductionCircuitToSpinGlass, ReductionMaxCutToSpinGlass, ReductionSpinGlassToMaxCut, ReductionVertexCoveringToSetCovering, ReductionSatToColoring
+export ReductionSATTo3SAT, rename_variables, transform_to_3_literal_clause, transform_to_3_literal_cnf, generate_dummy_var
```

Comment:
Some functions not exposed to users should not be exported.

Reply:
Resolved